src/chexus/__main__.py

```diff
@@ -65,6 +65,7 @@ def main():
     if _is_text_file(path):
         group = chexus.read_json(path)
     else:
+        # File is closed when 'reader' goes out of scope
```

Comment:
```suggestion
        # File is closed when 'reader' goes out of scope. We need to keep it open for lazily loading values.
```
... or what is the reason?